examples/speech-pretraining/run_pretraining.py

```diff
@@ -275,17 +275,26 @@ def __call__(self, features: List[Dict[str, Union[List[int], torch.Tensor]]]) ->
             min_masks=1,
         )
 
+        cropped_length = int(mask_indices_seq_length * self.model.config.mask_time_prob) + 1
+        # position true masked indexes first and crop, we mask the false indexes later
```

Comment:
# move true masked indexes first and crop, later used to gather a reduced size tensor.

I think this is a little better. Hard to word it nicely because it contains a small amount of the false positions too, but the true is first

Reply:
Yeah that's clearer.